Resources/Prototypes/_Lavaland/Entities/Structures/Specific/tendril.yml

```
@@ -120,6 +120,20 @@
     spawns:
     - MobCarpDungeon
 
+- type: entity
+  id: LavalandTendrilMagmaHand
+  parent: LavalandTendrilBase
+  name: tendril
+  description: A huge structure.
+  suffix: MagmaHand
+  components:
+  - type: Tendril
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)